src/imitation/scripts/train_preference_comparisons.py

```diff
@@ -74,17 +75,20 @@ def train_preference_comparisons(
     rl: Mapping[str, Any],
     allow_variable_horizon: bool,
     checkpoint_interval: int,
+    query_schedule: str,
```

Comment:
No need to restrict this to `str` -- we can define functions in Sacred configs. (Hacky admittedly, but may be necessary in some cases.)

Reply:
Interesting, good to know. `Union[str, Callable[[float], float]]` should work then.